### PR TITLE
Overview: Link to learning objectives as well as prep

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: common-theme/layouts/_default/overview.html [hunk 1]
```diff
@@ -23,27 +23,24 @@
...
-            {{ $block := ($site.GetPage .src) }}
-            {{ if or $block.Params.hide_from_overview (strings.Contains .src "https:") }}
-              {{ continue }}
-            {{ end }}
-            <li>{{ $block.Title }}</li>
-          {{ end }}
-        </ol>
+      {{ $prep := .Site.GetPage (printf "%sprep/index.md" .CurrentSection.RelPermalink) }}
+      {{ $success := .Site.GetPage (printf "%ssuccess/index.md" .CurrentSection.RelPermalink) }}
+      {{ if not $hasShownModuleTitle }}
```
Comment (SallyMcGrath) [inline common-theme/layouts/_default/overview.html:28]: Psst, just from reading, not checking out on local, it looks like you've removed the with. I think you still need with, both with prep and then further down with success. Is it on purpose?

(`[with](https://gohugo.io/functions/go-template/with/)` is the most reliable way of handling null safety in hugo)

Comment (illicitonion) [reply common-theme/layouts/_default/overview.html:28]: I just re-pushed this branch as 4 commits - the first three I _think_ are no-op refactors? (Particularly, if you view the third one with "hide whitespace")

I assume removing a `with` block is fine as long as nothing ever tries to use `.` directly? Or does it have some extra nil-related behaviour I don't know about?

Comment (SallyMcGrath) [reply common-theme/layouts/_default/overview.html:29]: Well I don't like to print empty blocks on the page, as a rule. I'd prefer to not show anything if there's no content in prep.

File: org-cyf-itp/content/overview/_index.md [hunk 2]
```diff
@@ -2,14 +2,13 @@
...
+emoji="🦉"
 +++
 
-## Objective
+### Objective
 
-The objective of this course is to train people with no prior software development experience to get a good job in technology.
-
-Primarily this is achieved by teaching programming concepts using JavaScript (both client-side and server-side).
+The objective of this course is to train people with no prior software development experience to get an entry level job in tech or IT. Primarily this is achieved by teaching programming concepts with JavaScript, Python, and data concepts with SQL.
```
Comment (illicitonion) [inline org-cyf-itp/content/overview/_index.md:10]:
```suggestion
The objective of this course is to give people with no prior software development experience an introduction to programming and thinking about data. Common next-steps after this course are to get an entry level job in tech or IT, or to continue learning more advanced software engineering techniques to prepare for a software engineering job.

Primarily this is achieved by teaching programming concepts with JavaScript, and data concepts with SQL.
```

**Review by SallyMcGrath — APPROVED** [review]
lol, approved whoops

**Pull request merged**